presto-docker/bin/run-presto

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+if [[ ! -d /usr/lib/presto/etc ]]; then
+    ln -s /usr/lib/presto/default/etc /usr/lib/presto/etc
+fi
+
+set +e
+grep 'node.id' /usr/lib/presto/etc/node.properties >/dev/null 2>&1
```

Comment:
You can use `grep -s -q` to suppress output

Reply:
👍